### PR TITLE
add frontend torch sigmoid function

### PR DESCRIPTION
**Pull request by Arcadia24**
Hi!

here my implementation to add torch.sigmoid.

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py [hunk 1]
```diff
@@ -2407,3 +2407,34 @@ def test_torch_sinc(
...
+@handle_frontend_test(
+    fn_tree="torch.sigmoid",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+)
+def test_torch_sigmoid(
+    *,
+    dtype_and_x,
+    as_variable,
```
Comment (zaeemansari70) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py:2421]: We had a small refactor, you have to refactor the tests according to this.
https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py#L29
Thanks!

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py (hunk 1 above)
Comment (Arcadia24) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py]: Done !

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Awesome! Your tests are passing. 🙂 
There are some lint errors, you just need to fix those so we can merge your PR 🙂 
Thanks!

**Comment by Arcadia24** [comment]
Format fixed for both functions ! 
I had format problems with the duplication_checking_1 file but I fix it to pass the check_formatting test👍

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Awesome! Thanks for the changes!

Actually your PR shows changes in the `duplicate_checking_1.py` file which is not part of the task 🙂 
Moreover, this file `duplicate_checking_1.py` is also removed from the main repo which is causing a conflict on your PR. 
Your PR should only show changes in 2 files only.

1.ivy/functional/frontends/torch/pointwise_ops.py
2. ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

Kindly resolve the conflict so we can merge your PR.
No other change needed afterwards🙂 
Thanks!

**Pull request closed without merging**